presto-verifier/src/main/java/com/facebook/presto/verifier/HumanReadableEventClient.java

```diff
@@ -59,7 +145,56 @@ private static void printEvent(VerifierQueryEvent queryEvent)
         else {
             out.println("Control Duration (secs): " + queryEvent.getControlWallTimeSecs());
             out.println("   Test Duration (secs): " + queryEvent.getTestWallTimeSecs());
+
+            Optional<Double> cpuRatio = getCpuRatio(queryEvent);
+            if (cpuRatio.isPresent()) {
+                out.println("Control CPU (secs): " + queryEvent.getControlCpuTimeSecs());
+                out.println("   Test CPU (secs): " + queryEvent.getTestCpuTimeSecs());
+                out.println(format("         CPU Ratio: %.1f\n", (double) cpuRatio.get()));
+            }
         }
         out.println("----------");
     }
+
+    private Optional<Double> getCpuRatio(VerifierQueryEvent queryEvent)
+    {
+        Double controlCpuTime = queryEvent.getControlCpuTimeSecs();
+        Double testCpuTime = queryEvent.getTestCpuTimeSecs();
+        if (controlCpuTime == null || testCpuTime == null) {
+            return Optional.empty();
+        }
+        if (controlCpuTime < regressionMinCpuTime.getValue(SECONDS)) {
+            return Optional.empty();
+        }
+
+        double value = testCpuTime / controlCpuTime;
+
+        // if both are 0.0, we had no speedup
+        if (isNaN(value)) {
+            value = 1.0;
+        }
+
+        return Optional.of(value);
+    }
+
+    /**
+     * Converts a double value to a sortable long. The value is converted by getting their IEEE 754
+     * floating-point bit layout. Some bits are swapped to be able to compare the result as long.
+     */
+    public static double sortableLongToDouble(long value)
```

Comment:
Also, it might be easier to understand if you just converted to milliseconds and stored that in the long. We probably don't need the full precision of the double